src/tbb/market.cpp

```diff
@@ -203,7 +203,7 @@ bool market::release ( bool is_public, bool blocking_terminate ) {
             theMarket = NULL;
         }
     }
-    if( do_release ) {
+    if ( do_release ) {
```

Comment:
```suggestion
    if( do_release ) {
```

Reply:
No, check another codestyle in this file https://github.com/oneapi-src/oneTBB/blob/master/src/tbb/market.cpp#L222